clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListCatalogs.java

```diff
@@ -49,7 +49,11 @@ public void handle() {
     try {
       GravitinoClient client = buildClient(metalake);
       catalogs = client.listCatalogsInfo();
-      output(catalogs);
+      if (catalogs.length == 0) {
+        System.out.println("No catalogs exists.");
+      } else {
+        output(catalogs);
+      }
```

Comment:
It is odd to call `System.out.println` in one branch and `output` in another, unless there is an implicit protocol.

Reply:
![image](https://github.com/user-attachments/assets/aad92828-e336-4530-9577-e3c67679492e)

That logic is being written by another member whose code is not complete. Strings cannot be displayed using this method.